cmd/cli/version/version.go

```diff
@@ -30,21 +34,24 @@ import (
 	"github.com/bacalhau-project/bacalhau/cmd/util"
 	"github.com/bacalhau-project/bacalhau/cmd/util/flags/cliflags"
 	"github.com/bacalhau-project/bacalhau/cmd/util/output"
+
+	"github.com/bacalhau-project/bacalhau/pkg/config"
 )
 
-// Versions is a struct for version information
 type Versions struct {
-	ClientVersion *models.BuildVersionInfo `json:"clientVersion,omitempty"`
-	ServerVersion *models.BuildVersionInfo `json:"serverVersion,omitempty"`
+	ClientVersion  *models.BuildVersionInfo `json:"clientVersion,omitempty"`
+	ServerVersion  *models.BuildVersionInfo `json:"serverVersion,omitempty"`
+	OperatingSystem string                   `json:"operatingSystem,omitempty"`
+	Architecture    string                   `json:"architecture,omitempty"`
+	UserID          string                   `json:"userID,omitempty"`
```

Comment:
What is the point of adding these fields to the struct? They are set but never read from anywhere? Plus, the `BuildVersionInfo` struct already contains OS and ARch fields:
https://github.com/bacalhau-project/bacalhau/blob/f1a8337869a6a947b5685a1297d6a505800ed57b/pkg/models/buildversion.go#L18-L19

How about instead you just add a `LatestVersion` field? And make the server return a `BuildVersionInfo` struct?
```suggestion
	LatestVersion   *models.BuildVersionInfo `json:"latestVersion,omitempty"`
```